blockchain/pool.go

```diff
@@ -54,6 +54,9 @@ func (e *emulatorPool) new() (*emulator, error) {
 		return em, nil
 	default: // in case pool gets emptied
 		sentry.CaptureMessage("instance pool empty")
+		for i := 0; i < cap(e.instances); i++ {
```

Comment:
this should not be needed as there are already routines spawned that will fill the pool since every instances removed also spawns a routine creating an instance, this will only happen on huge influx of lots of instance requests but eventually will be filled again. Think of it as a buffer

Reply:
Because this will start being a subroutine inside subroutines etc. Should be fine but just harder to understand

Reply:
Okay will remove for now. We could also refresh the pool once per day or something instead later.

Reply:
It should be fine tho, this error is not expected, it could even be said it's a panic level error, so we haven't really experienced it. But we can keep an eye on this.